src/olympia/discovery/admin.py

```diff
@@ -32,7 +32,8 @@ class Media:
         css = {
             'all': ('css/admin/discovery.css',)
         }
-    list_display = ('__unicode__', 'custom_addon_name', 'custom_heading',)
+    list_display = ('__unicode__', 'custom_addon_name', 'custom_heading',
+                    'position', 'position_china')
     raw_id_fields = ('addon',)
```

Comment:
Also need a `list_filter` to allow admins to quickly find add-ons that do have a position set.